docs/source/model_definition.md

```diff
@@ -33,28 +68,43 @@ tabular_inputs = TabularSequenceFeatures.from_schema(
     )
 ```
 
-
 ## Sequence Masking
-Transformer architectures can be trained in different ways. Depending on the training method, there is a specific masking schema. The masking schema sets the items to be predicted (labels) and masks some positions of the sequence that cannot be used by the Transformer layers for prediction. Transformers4Rec currently supports the following training approaches, inspired by NLP:
 
-- **Causal LM (`masking="clm"`)** - Predicts the next item based on past positions of the sequence. Future positions are masked.
-- **Masked LM (`masking="mlm"`)** - Randomly select some positions of the sequence to be predicted, which are masked. The Transformer layer is allowed to use positions on the right (future information) during training. During inference, all past items are visible for the Transformer layer, which tries to predict the next item.
-- **Permutation LM (`masking="plm"`)** - Uses a permutation factorization at the level of the self-attention layer to define the accessible bidirectional context
-- **Replacement Token Detection (`masking="rtd"`)** - Uses MLM to randomly select some items, but replaces them by random tokens. Then, a discriminator model (that can share the weights with the generator or not), is asked to classify whether the item at each position belongs to the original sequence. The generator-discriminator architecture is jointly trained using Masked LM and RTD tasks.
+You can train Transformer architectures in different ways.
+Depending on the training method, there is a specific masking schema.
+The masking schema sets the items to be predicted--labels--and masks some positions of the sequence that cannot be used by the Transformer layers for prediction.
+
+Transformers4Rec supports the following training approaches that are inspired by NLP:
+
+- **Causal Language Modeling (`masking="clm"`)**: Predicts the next item based on past positions of the sequence.
+Future positions are masked.
+
+- **Masked Language Modeling (`masking="mlm"`)**: Randomly selects some positions of the sequence to predict, which are masked.
+The Transformer layer is allowed to use positions on the right--future information--during training.
+During inference, all past items are visible for the Transformer layer as it tries to predict the next item.
+
+- **Permutation Language Modeling (`masking="plm"`)**: Uses a permutation factorization at the level of the self-attention layer to define the accessible bi-directional context.
 
-Note that not all transformer architectures support all of these training approaches.  Transformers4Rec will raise an exception when you attempt to use an invalid combination and will provide suggestions as to the appropriate masking techniques for that architecture.
+- **Replacement Token Detection (`masking="rtd"`)**: Uses MLM to randomly select some items, but replaces them by random tokens.
```

Comment:
Currently, the `rtd` is not supported out of the box (unlike the three others) and would require the user to define a custom T4Rec block to be able to use it. So we could maybe remove it from this list?